### PR TITLE
docs: reword providers section

### LINKED ISSUE #134: Reword "supported providers" section
# Description

The word `supported` may cause misunderstanding as a customer. We need to find and replace the word in the docs, potentially using `validated`|`verified` providers.

### PR DESCRIPTION
**Pull request by salasberryfin**
# Description

After discussions with the team and product, we agreed that we should substitute the term "supported" from the documentation and reflect that the providers are verified/validated instead.

#### Fixes #134

### CODE REVIEW

File: docs/reference-guides/providers/howto.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ import TabItem from '@theme/TabItem';
 
 # Create & import a cluster using CAPI providers
 
-This guide goes over the process of creating and importing CAPI clusters with a selection of the officially supported providers.
+This guide goes over the process of creating and importing CAPI clusters with a selection of the officially verified providers.
```
Comment (furkatgofurov7) [inline docs/reference-guides/providers/howto.md:10]: Doing some nitpicking however why not use same wording everywhere?

Comment (salasberryfin) [reply docs/reference-guides/providers/howto.md:10]: Thanks @furkatgofurov7. I agree the term must be the same to be clear. I just updated this.

**Review by furkatgofurov7 — APPROVED** [review]

**Review by Danil-Grigorev — COMMENTED** [review]
Can this also be called `certified` providers? We can expand on the certification process later.

**Comment by alexander-demicev** [comment]
@Danil-Grigorev makes sense to me, I already opened an issue about this https://github.com/rancher/turtles-docs/issues/136

**Comment by salasberryfin** [comment]
No preference on what term to use from my side @Danil-Grigorev. Maybe we need to confirm if certified is a valid alternative for product?

File: docs/reference-guides/providers/validated.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ This is a list of the officially supported CAPI Providers by Turtles. These prov
 
 ## List of providers in experimental mode
 
-This is a list of providers that are in an advanced state of development and will soon become officially supported.
+This is a list of providers that are in an advanced state of development and will soon become verified.
```
Comment (kkaempf) [inline docs/reference-guides/providers/validated.md:28]:
```suggestion
This is a list of providers that are in an advanced state of development and will soon become validated.
```

**Review by kkaempf — CHANGES_REQUESTED** [review]

File: docs/reference-guides/providers/validated.md (hunk 2 above)
Comment (kkaempf) [reply docs/reference-guides/providers/validated.md:28]: We should not have mixed wording of validated and verified.

**Comment by salasberryfin** [comment]
Changed to the proposed `certified` so that we can also work on a certification process and document how a provider becomes certified.

**Review by Danil-Grigorev — DISMISSED** [review]

**Review by furkatgofurov7 — DISMISSED** [review]

**Review by Danil-Grigorev — APPROVED** [review]

**Review by mjura — DISMISSED** [review]

File: docs/reference-guides/providers/howto.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ import TabItem from '@theme/TabItem';
 
 # Create & import a cluster using CAPI providers
 
-This guide goes over the process of creating and importing CAPI clusters with a selection of the officially supported providers.
+This guide goes over the process of creating and importing CAPI clusters with a selection of the officially validated providers.
```
Comment (yiannistri) [inline docs/reference-guides/providers/howto.md:10]: Nit: perhaps we should also use the term `certified` here.

**Review by yiannistri — APPROVED** [review]

**Review by yiannistri — APPROVED** [review]

**Review by alexander-demicev — APPROVED** [review]

**Review by Danil-Grigorev — APPROVED** [review]

**Review by kkaempf — APPROVED** [review]
lgtm

**Pull request merged**